src/test/kotlin/org/rust/ide/refactoring/RenameTest.kt

```diff
@@ -11,7 +11,12 @@ class RenameTest: RustTestCaseBase() {
     private fun doTest(name: String = "spam") {
         myFixture.configureByFile(fileName)
         myFixture.renameElementAtCaret(name)
-        myFixture.checkResultByFile(fileName, fileName.replace(".rs", "_after.rs"), false);
+        myFixture.checkResultByFile(fileName, fileName.replace(".rs", "_after.rs"), false)
+    }
+
+    fun testRenameFile() = checkByDirectory {
```

Comment:
@matklad the NPE test is added, as the reason to spur the NPE is that if we rename `foo.rs` if we have usage of `foo` module in other file.